extensions/Dockerfile

```
@@ -24,6 +24,8 @@ RUN if [[ -n "${VARIANT}" ]]; then MANIFEST="manifest-${VARIANT}.yaml"; EXTENSIO
 ## current p8/s390x. See https://github.com/openshift/os/issues/1000
 FROM quay.io/fedora/fedora:40 as builder
 COPY --from=os /usr/share/rpm-ostree/extensions/ /usr/share/rpm-ostree/extensions/
+RUN rm -f /etc/yum.repos.d/*.repo \
+&& curl -L https://raw.githubusercontent.com/coreos/fedora-coreos-config/testing-devel/fedora.repo -o /etc/yum.repos.d/fedora.repo
```

Should this be changed for the older branches?

```suggestion
&& curl -L https://raw.githubusercontent.com/coreos/fedora-coreos-config/rhcos-4.17/fedora.repo -o /etc/yum.repos.d/fedora.repo

---

I did this anyway. It makes sense to use the older `rhcos-4.xx` branches that are in sync with the older releases.